test/ILLink.Tasks.Tests/Mock.cs

```diff
@@ -20,6 +20,7 @@ public MockTask ()
 			// Ensure that [Required] members are non-null
 			AssemblyPaths = new ITaskItem[0];
 			RootAssemblyNames = new ITaskItem[0];
+			ILLinkPath = Path.Combine (Path.GetDirectoryName (Assembly.GetExecutingAssembly ().Location), "illink.dll");
```

Comment:
Curious what prompted this change

Reply:
The login in the ILLink is basically `MyPath/../../bin/net5.0/illink.dll`. Which worked before by luck, since the folder in the artifacts where the test lives ends with bin/net5.0 as well. Now when the test is running on 6, the path ends with bin/net6.0.
But the product is still NET5.